.circleci/config.yml

```diff
@@ -1,9 +1,20 @@
 version: 2.1
 
 executors:
-  default:
+  build:
     docker:
       - image: circleci/golang:1.13
+  deploy:
+    docker:
+      - image: circleci/node:8.10
```

Comment:
Are there docker images that bring `serverless` with them?

Reply:
I tried looking for a while, but admit to not being able to find one. Definitely would prefer that!

Reply:
Does https://hub.docker.com/r/softinstigate/serverless work?

Reply:
Possibly? I admit I don't know how to "trust" Docker hub images. I feel like I can trust `circleci` (since I already have to), but should I do the same for `softinstigate`?